image/nitf.ksy

```diff
@@ -0,0 +1,759 @@
+      - id: actual_bits_per_pixel_per_band
+        -orig-id: abpp
+        type: str
+        size: 2
+      - id: pixel_justification
+        -orig-id: pjust
+        type: str
+        size: 1
+      - id: image_coordinate_rep
+        -orig-id: icords
+        type: str
+        size: 1
+      - id: image_geo_loc
+        -orig-id: igeolo
+        type: str
+        size: 60
+      - id: num_img_comments
+        -orig-id: nicom
+        type: str
+        size: 1
+      - id: img_comments
+        -orig-id: icom
+        type: image_comment
+        repeat: expr
+        repeat-expr: num_img_comments.to_i
+      - id: img_compression
+        -orig-id: ic
+        type: str
+        size: 2
+      - id: compression_rate_code
+        -orig-id: comrat
+        type: str
+        size: 4
+      - id: num_bands
+        -orig-id: nbands
+        type: str
+        size: 1
+      - id: num_multispectral_bands
+        -orig-id: xbands
+        type: str
+        size: 5
+        if: num_bands.to_i == 0
+      - id: bands
+        type: band_info
+        repeat: expr
+        repeat-expr: "num_bands.to_i != 0 ? num_bands.to_i : num_multispectral_bands.to_i"
+      - id: img_sync_code
+        -orig-id: isync
+        type: str
+        size: 1
+        doc: 'Reserved for future use.'
+      - id: img_mode
+        -orig-id: imode
+        type: str
+        size: 1
+        doc: 'B = Band Interleaved by Block, P = Band Interleaved by Pixel, R = Band Interleaved by Row, S = Band Sequential'
+      - id: num_blocks_per_row
+        -orig-id: nbpr
+        type: str
+        size: 4
+      - id: num_blocks_per_col
+        -orig-id: nbpc
+        type: str
+        size: 4
+      - id: num_pixels_per_block_horz
+        -orig-id: nppbh
+        type: str
+        size: 4
+      - id: num_pixels_per_block_vert
+        -orig-id: nppbv
+        type: str
+        size: 4
+      - id: num_pixels_per_band
+        -orig-id: nbpp
+        type: str
+        size: 2
+      - id: img_display_level
+        -orig-id: idlvl
+        type: str
+        size: 3
+      - id: attachment_level
+        -orig-id: ialvl
+        type: str
+        size: 3
+      - id: img_location
+        -orig-id: iloc
+        type: str
+        size: 10
+      - id: img_magnification
+        -orig-id: imag
+        type: str
+        size: 4
+      - id: user_def_img_data_len
+        -orig-id: udidl
+        type: str
+        size: 5
+      - id: user_def_overflow
+        -orig-id: udofl
+        type: str
+        size: 3
+        if: user_def_img_data_len.to_i != 0
+      - id: user_def_img_data
+        -orig-id: udid
+        type: u1
+        if: user_def_img_data_len.to_i > 2
+        repeat: expr
+        repeat-expr: user_def_img_data_len.to_i - 3
+      - id: image_extended_sub_header
+        type: tre_header
+  band_info:
+    seq:
+      - id: n_band_representation

```

Comment:
should `n_band_` be ommitted?

Reply:
I had it in there per the doc naming (pg 97 of reference), but have removed.